packages/sourcecred/package.json

```diff
@@ -186,7 +186,8 @@
     "/dist",
     "/src",
     "!src/**/*.test.js?(.snap)",
-    "!src/ui"
+    "!src/ui",
+    "!src/**/snapshots"
```

Comment:
do we need to include `!src/**/__snapshots__` too? Or are the underscores some weird path symbol I don't understand

Reply:
The optional .snap trailer excludes the files in the __snapshots__ folders

Reply:
ah i see